draft-ietf-quic-transport.md

```diff
@@ -239,6 +228,24 @@ Server:
 
 : The endpoint that accepts a QUIC connection.
 
+QUIC packet:
+
+: A complete processable unit of QUIC that can be encapsulated in a UDP
+  datagram.  One or more QUIC packets can be encapsulated in a single UDP
+  datagram.
+
+Ack-eliciting Packet:
+
+: A QUIC packet that contains frames other than ACK, PADDING, and
+  CONNECTION_CLOSE. These cause a recipient to send an acknowledgment; see
+  {{sending-acknowledgements}}.
+
+Frame:
+
+: A unit of structured protocol information.  There are multiple frame types,
+  each of which carries different information.  Frames are conveyed in QUIC
```

Comment:
```suggestion
: A unit of structured protocol information.  A frame begins with a frame type,
  which indicates what information is contained in the frame. Frames are conveyed in QUIC
```

Reply:
I think we can skip that.  It's correct, but I don't think we need it here.

Reply:
I agree; that's too detailed for the definitions list.  I think the furthest I'd go is to say there are multiple types of frames, but I'm not sure we need even that.